test/components/masterDetailsComponent.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { useMasterDetails } from '../../src/useMasterDetails';
+
+export const MasterDetailsComponent = () => {
+    const [open, openDetails] = useMasterDetails();
+    return (
+        <>
```

Comment:
You dont need a wrapper if you have div. Keep fragment, it's better

Reply:
done